compiler/codegen/cg-expr.cpp

```diff
@@ -4928,6 +4922,10 @@ static GenRet codegenGPUKernelLaunch(CallExpr* call, bool is3d) {
   }
 
   return codegenCallExprWithArgs(fn, args);
+#else
+  INT_FATAL("Unexpected code path: gpu code generation without LLVM as target");
+  return NULL;
```

Comment:
Nit: (granted this codepath should never execute) but we should use `nullptr` instead of `NULL` when possible.